spec/features/hiring_staff_can_view_vacancies_spec.rb

```diff
@@ -4,6 +4,15 @@
   include_context 'when authenticated as a member of hiring staff',
                   stub_basic_auth_env: true
 
+  scenario 'A school should see advisory text when there are no vacancies', elasticsearch: true do
+    school = FactoryGirl.create(:school)
+    visit school_path(school.id)
+
+    expect(page).to have_content(I18n.t('schools.vacancies.index', school: school.name))
+    expect(page).not_to have_css('table.vacancies')
+    expect(page).to have_content(I18n.t('schools.no_vacancies.heading'))
```

Comment:
For expectations, it would be better if use the actual expected text, as lack of verifying content was what caused me to override the application_link.

Reply:
You think that's preferable to having the text in one place only?

Reply:
Since this is specifically about the content you see when you get to that page yes, I don't class this it's duplication.